CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 message(STATUS "Using CMake ${CMAKE_VERSION}")
-cmake_minimum_required(VERSION 3.19.0)
+cmake_minimum_required(VERSION 3.20.0)
```

Comment:
using `cmake_path` which is new in 3.20

https://cmake.org/cmake/help/latest/command/cmake_path.html